CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Circular dependency
+
+This package has a development dependency on `doctrine/common`, which has a
```

Comment:
Thanks for documenting this. Is there anything we can do to fix this? Should we prepare doctrine/common to work with persistence 3?

Reply:
We could use `^2.0 || ^3.0`, but IMO that's a bit "risky" if we do not test that 2.0 actually work with 3, and the benefit is small: that would only fix one weird thing out of the 2 we have to do, namely:
- using `COMPOSER_ROOT_VERSION`
- using `2.0` as a value for it when developing on 3.x